wgpu-native/src/device.rs

```diff
@@ -60,6 +60,9 @@ const CLEANUP_WAIT_MS: u64 = 5000;
 pub const MAX_COLOR_TARGETS: usize = 4;
 pub const MAX_VERTEX_BUFFERS: usize = 8;
 
+/// Descriptor uniform/storage buffer offsets must be aligned to this number.
+pub const DESCRIPTOR_BUFFER_OFFSET_ALIGN: hal::buffer::Offset = 256;
```

Comment:
let's rename to `BIND_BUFFER_ALIGNMENT`, first - because "descriptor" is not an API term in WebGPU, and second - because there should be no need to shorten "ALIGN"

Reply:
Yeah makes sense.